src/state/nftMarket/helpers.ts

```diff
@@ -38,7 +39,8 @@ interface NftsFromSubgraph {
   isTradable: boolean
   metadataUrl: string
 }
-export const getNftsMarketData = async (collectionAddress: string): Promise<NftsFromSubgraph[]> => {
+
+export const getNftsFromCollection = async (collectionAddress: string): Promise<NftsFromSubgraph[]> => {
```

Comment:
Rename to more accurately reflect the request

Reply:
There is a reason why it was called `getNftsMarketData` and not `getNftsFromCollection`. This call only fetch data from the subgraph (i.e market data), nft data will come from the api

Reply:
Hmmn yea I hear ya. Naming 20 different references to NFTs and Collections and Markets is going to be tricky, and may need continued tweaking. I'll change this.

Reply:
Renamed to `getNftCollectionMarketData`